src/test/resources/mapping/expectedTransformationFields.json

```diff
@@ -1,231 +1,257 @@
 {
-  "transformationFields": [
```

Comment:
In the PR for MDEXP-217, I added tests  that check the equals of expected and actual response, to check the order of the fields,, but let`s leave it as it is here, I will modify the file again and commit in my PR